pkg/api/daemon.go

```diff
@@ -53,10 +56,31 @@ func Run(context *clusterd.Context, c *Config) error {
 		return fmt.Errorf("failed to write connection config. %+v", err)
 	}
 
+	CephMonConfWatcher(context, c)
 	ServeRoutes(context, c)
 	return nil
 }
 
+func CephMonConfWatcher(context *clusterd.Context, c *Config) {
```

Comment:
This looks great, nice and simple. I might suggest a couple minor things:
- Name this method `WatchMonConfig`
- Instead of the goroutine inside the method, call `go WatchMonConfig(context, c)`. I prefer the pattern that the caller can choose if it should be a goroutine, but what you have works perfectly fine too of course.